src/frontend/src/utils/iiConnection.ts

```diff
@@ -73,10 +76,20 @@ if (typeof canisterId !== undefined) {
 }
 
 export const canisterIdPrincipal: Principal = Principal.fromText(canisterId);
-export const baseActor = Actor.createActor<_SERVICE>(internet_identity_idl, {
-  agent: new HttpAgent({}),
-  canisterId,
-});
+
+async function createBaseActor(): Promise<ActorSubclass<_SERVICE>> {
```

Comment:
as far as I can tell, the only difference between this and `createActor()` (line 349) is that is basically `createActor(null)`; is there a reason for having both? Did I miss a subtlety?

Reply:
No, you're right. I initially stumbled across this problem, because the baseActor did not fetch root keys correctly. But the fix I made was of course totally redundant to the existing createActor() function.